templates/lobbys/lobby.html

```diff
@@ -177,10 +211,45 @@ <h2 v-else>Lobby [[lobbyInfo.lobby_id]]: </h2>
         <span id="custom-tooltip" ref="shareTooltip">Copied invite to clipboard!</span>
     </div>
 
+    
 
     <template v-if="lobbyInfo.user.owner">
+
         <hr>
-        <h4>Manage: Add a prompt</h4>
+
+        <h3>Lobby settings</h4>
+
+        <details>
+            <summary>List of players</summary>
+            <div class="card">
+                <div class="card-body">
+                    <table class="table table-hover">
+                        <thead>
+                            <th scope="col">Display name</th>
+                            <th scope="col">Host</th>
```

Comment:
Is the host column necessary? I can see this being useful if we want to support multiple hosts in the future, but right now it seems like we just support one, in which case I think the column is confusing to have in the UI (since only the host sees this anyways).

Related - I think "Make host" is not clear right now, since with the Host column, it feels like you are _adding_ a new host rather than _transferring_ ownership. I think we should either 
1. change the functionality here to add an additional host
  1a. Change the backend/db schema to support a set of userid's rather than a single userid
2. change the text to reflect the transfer i.e. "Make host" --> "Transfer Ownership" or similar and make that clear in the pop-up as well.